plugin-server/src/worker/ingestion/ingest-event.ts

```diff
@@ -60,3 +62,48 @@ export async function ingestEvent(hub: Hub, event: PluginEvent): Promise<IngestE
         clearTimeout(timeout)
     }
 }
+
+export async function ingestBufferEvent(hub: Hub, event: PreIngestionEvent): Promise<IngestEventResponse> {
+    const person = await hub.db.fetchPerson(event.teamId, event.distinctId)
```

Comment:
@tiina303 essentially I'd need to look at all the code paths and see what they use from person but we might be able to substitute this for a cached version